html/pfappserver/root/static.alt/src/views/Users/index.vue

```diff
@@ -49,12 +49,12 @@ export default {
           name: 'Standard Searches',
           items: [
             {
-              name: 'Open Violations',
-              path: 'search/openviolations'
+              name: 'Open Security Events',
+              path: 'search/opensecurity_events'
```

Comment:
I known those are not yet implemented, but let's fix this path using snake case: `search/open_security_events`.

Reply:
The paths would need to be also adjusted in the routing in pf::UnifiedApi.

Reply:
We could do that after merging.